lib/config/paris-config.ts

```diff
@@ -1,11 +1,12 @@
-import {AjaxRequest} from "rxjs/ajax";
+import {ajax, AjaxRequest} from "rxjs/ajax";
 
 export interface ParisConfig<TConfigData = any>{
 	apiRoot?:string,
 	allItemsProperty?:string,
 	entityIdProperty?:string,
 	data?:TConfigData,
-	http?:AjaxRequest
+	http?:AjaxRequest,
+	ajaxService?: typeof ajax
```

Comment:
`ajaxService` should be a property of the `Http` class, since it's an implementation detail of that class. Please set it in the constructor of `Http` (you'll have to make the methods non-static).